verticapy/tests/vModel/test_normalizer.py

```diff
@@ -0,0 +1,231 @@
+
+    def test_drop(self, base):
+        base.cursor.execute("DROP MODEL IF EXISTS norm_model_test_drop")
+        model_test = Normalizer("norm_model_test_drop", cursor=base.cursor)
+        model_test.fit("public.winequality", ["alcohol", "quality"])
+
+        base.cursor.execute(
+            "SELECT model_name FROM models WHERE model_name = 'norm_model_test_drop'"
+        )
+        assert base.cursor.fetchone()[0] == "norm_model_test_drop"
+
+        model_test.drop()
+        base.cursor.execute(
+            "SELECT model_name FROM models WHERE model_name = 'norm_model_test_drop'"
+        )
+        assert base.cursor.fetchone() is None
+
+    def test_get_attr(self, model):
+        m_att = model.get_attr()
+
+        assert m_att["attr_name"] == [
+            "details",
+        ]
+        assert m_att["attr_fields"] == [
+            "column_name, avg, std_dev",
+        ]
+        assert m_att["#_of_rows"] == [3]
+
+        m_att_details = model.get_attr(attr_name="details")
+
+        assert m_att_details["column_name"] == [
+            "citric_acid",
+            "residual_sugar",
+            "alcohol",
+        ]
+        assert m_att_details["avg"][0] == pytest.approx(0.318633215330152, abs=1e-6)
+        assert m_att_details["avg"][1] == pytest.approx(5.44323533938741, abs=1e-6)
+        assert m_att_details["avg"][2] == pytest.approx(10.4918008311528, abs=1e-6)
+        assert m_att_details["std_dev"][0] == pytest.approx(0.145317864897592, abs=1e-6)
+        assert m_att_details["std_dev"][1] == pytest.approx(4.75780374314742, abs=1e-6)
+        assert m_att_details["std_dev"][2] == pytest.approx(1.192711748871)
+
+    def test_get_params(self, model):
+        assert model.get_params() == {'method': 'zscore'}
+
+    def test_to_sklearn(self, model):
+        # Zscore
+        md = model.to_sklearn()
+        model.cursor.execute(
+            "SELECT APPLY_NORMALIZE(citric_acid, residual_sugar, alcohol USING PARAMETERS model_name = '{}', match_by_pos=True) FROM (SELECT 3.0 AS citric_acid, 11.0 AS residual_sugar, 93. AS alcohol) x".format(
+                model.name
+            )
+        )
+        prediction = model.cursor.fetchone()[0]
+        assert prediction == pytest.approx(md.transform([[3.0, 11.0, 93.0]])[0][0])
+        # Minmax
+        model2 = Normalizer("norm_model_test2", cursor=model.cursor, method = "minmax")
+        model2.drop()
+        model2.fit(
+            "public.winequality", ["citric_acid", "residual_sugar", "alcohol"]
+        )
+        md = model2.to_sklearn()
+        model2.cursor.execute(
+            "SELECT APPLY_NORMALIZE(citric_acid, residual_sugar, alcohol USING PARAMETERS model_name = '{}', match_by_pos=True) FROM (SELECT 3.0 AS citric_acid, 11.0 AS residual_sugar, 93. AS alcohol) x".format(
+                model2.name
+            )
+        )
+        prediction = model2.cursor.fetchone()[0]
+        assert prediction == pytest.approx(md.transform([[3.0, 11.0, 93.0]])[0][0])
```

Comment:
A good written test should clean up after itself.
Please drop model2 when you don't need it anymore.

Reply:
Done;